lib/p5.play.js

```diff
@@ -4389,10 +4389,10 @@ p5.prototype._warn = function(message) {
     var radiusOnX = this._getRadiusOnAxis(p5.CollisionShape.X_AXIS);
     var radiusOnY = this._getRadiusOnAxis(p5.CollisionShape.Y_AXIS);
     return {
-      top: this.center - radiusOnY,
-      bottom: this.center + radiusOnY,
-      left: this.center - radiusOnX,
-      right: this.center + radiusOnX,
+      top: this.center.y - radiusOnY,
+      bottom: this.center.y + radiusOnY,
+      left: this.center.x - radiusOnX,
+      right: this.center.x + radiusOnX,
```

Comment:
This one's a real mess!  `this.center` is a `p5.Vector` object, while `radiusOnX/Y` is a number.  This resulted in `p5.Vector.prototype.toString` getting called four times for each call to `CollisionShape.prototype.getBoundingBox`, which - of course - is only used by the quadtree.  So not only was this terribly slow, it was also broken.  See the test I added below to make sure this method at least returns _numbers_ for the bounding box dimensions.

I tried enabling the quadtree again after fixing this since it's probably a huge part of the quadtree slowness - but it looks like our demo app still runs slower with the quadtree than without, and there's more work to do optimizing `getRadiusOnAxis`, etc. before we should turn it back on.